### ISSUE TITLE
Support custom threshold and recurrence on the API level 

### ISSUE THREAD

**Issue by drubin** [louy/terraform-provider-uptimerobot#14]
Currently, the scheme supports passing a custom threshold and recurrence information for each notification. It just doesn't seem to propagate it in the API requests.

The [following code](https://github.com/louy/terraform-provider-uptimerobot/blob/master/uptimerobot/api/monitor.go#L161-L165) Seems to be missing the information like the official doc.

https://uptimerobot.com/api#newMonitorWrap 
```
alert_contacts - optional (the alert contacts to be notified when the monitor goes up/down.Multiple alert_contact>ids can be sent like alert_contacts=457_0_0-373_5_0-8956_2_3 where alert_contact>ids are seperated with - and threshold + recurrence are seperated with _. For ex: alert_contacts=457_5_0 refers to 457 being the alert_contact>id, 5 being the threshold and 0 being the recurrence. As the threshold and recurrence is only available in the Pro Plan, they are always 0 in the Free Plan)
```

https://github.com/louy/terraform-provider-uptimerobot/blob/master/uptimerobot/api/monitor.go#L117 
https://github.com/louy/terraform-provider-uptimerobot/blob/master/uptimerobot/resource_uptimerobot_monitor.go#L87

**Comment by drubin** [comment]
I am working on a fix will submit a PR tomorrow if I manage to get it working.

**Comment by drubin** [comment]
Sorry, I totally forgot about this will try to get to it in the next week or so.

**Comment by drubin** [comment]
Sorry for leaving this off for a bit See #18

**Comment by louy** [comment]
Closed in #18. Thanks @drubin :)

**Comment by drubin** [comment]
@louy pleasure! thanks for reviewing it.
